circle.yml

```diff
@@ -24,7 +24,7 @@ general:
 
 deployment:
   release:
-    tag: /(([a-z]+)-)?([0-9]+)\.([0-9]+)\.([0-9]+)/
+    tag: /(([a-z]+)-)*([0-9]+)\.([0-9]+)\.([0-9]+)/
```

Comment:
> Some expected tags are in my 2nd comment. The reason we allow it to be empty is because umbrella tags are expected to be 0.x.y (discussed in #2810)

OK, I missed that this could be applied to the umbrella tags.  What does the script produce when the `CIRCLE_TAG` variable is set to an umbrella tag?

> Where did you imagine the documenting happens? Here or in another file?